sklearn/mixture/gaussian_mixture.py

```diff
@@ -563,6 +553,9 @@ class GaussianMixture(BaseMixture):
 
     n_iter_ : int
         Number of step used by the best fit of EM to reach the convergence.
+
+    lower_bound_ : float
```

Comment:
In fact for GMM this is the best log likelihood but not for VBGMM which is lower bound.
I've chosen lower_bound_ because it was the most understandable.